WordPress/Classes/ViewRelated/Site Creation/Web Address/AddressTableViewCell.swift

```diff
@@ -0,0 +1,317 @@
+import UIKit
+import WordPressKit
+
+final class AddressTableViewCell: UITableViewCell {
+
+    // MARK: - Dependencies
+
+    private let domainPurchasingEnabled = FeatureFlag.siteCreationDomainPurchasing.enabled
+
+    // MARK: - Views
+
+    private var borders = [UIView]()
+
+    private let domainLabel: UILabel = {
+        let label = UILabel()
+        label.adjustsFontForContentSizeCategory = true
+        label.font = Appearance.domainFont
+        label.textColor = Appearance.domainTextColor
+        return label
+    }()
+
+    private let tagsLabel: UILabel = {
+        let label = UILabel()
+        label.adjustsFontForContentSizeCategory = true
+        label.font = Appearance.tagFont
+        return label
+    }()
+
+    private let dotView: UIView = {
+        let size = Appearance.dotViewRadius * 2
```

Comment:
Quite minor but, I think we should call this something like `length` as size infers a 2-dimensional information.

Reply:
Resolved in https://github.com/wordpress-mobile/WordPress-iOS/pull/20344/commits/83fa2a3f369618eddd7f6ab45fa876fcc79ba41e